### DISCUSSION TITLE
GAMMA in xticks, how to fix it to Greek alphabet style

### DISCUSSION THREAD

**Discussion by 1lifetime1** [materialsproject/pymatgen#4025] [Q&A]
![py_error](https://github.com/user-attachments/assets/9c60fb0e-0e66-4deb-8068-9c1f3b2d22ac)

**Comment by DanielYang59** [comment]
Hi @1lifetime1 thanks for reporting this. Looks like a bug, or at least a possible improvement for the `BSDOSPlotter`. 

Is it possible for you to share the code snippet and data (if possible) to recreate this issue? Thanks a lot!

**Comment by phyfang** [comment]
Maybe you can change GAMMA to \Gamma in KPOINTS
